airflow/providers/google/cloud/transfers/bigquery_to_gcs.py

```diff
@@ -193,7 +193,7 @@ def _submit_job(
 
         return hook.insert_job(
             configuration=configuration,
-            project_id=hook.project_id,
+            project_id=configuration["extract"]["sourceTable"]["projectId"],
```

Comment:
@Yaro1  May I know the reason why we hard code the project id from sourceTable? We got the issue when we try to extract data from Project A but we want to submit job by using our own Project B. This line does not allow us to use our default project id.